docs/tbdev_getting_started.ipynb

```diff
@@ -1,36 +1,25 @@
 {
```

Comment:
<p>The change seems identical except for the newline. Revert the change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/2992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This the the format as output by Cider.  I'm reluctant to go through and manually override.

Reply:
<p>Is there a way to use ReviewNB to selectively revert diff blocks?  Can ReviewNB author commits?</p>

Reply:
> This the the format as output by Cider. I'm reluctant to go through and manually override.

I think the kosher way to edit these is to use Jupyter or Colab to for the right serialization and right output cell cache content.